code/modules/projectiles/ammunition/energy.dm

```diff
@@ -68,20 +68,28 @@
 /obj/item/ammo_casing/energy/immolator
 	projectile_type = /obj/item/projectile/beam/immolator
 	fire_sound = 'sound/weapons/laser3.ogg'
-	e_cost = 125
+	e_cost = 100 // 10 shots for the gamma sec ERT who get this, used to be 8
```

Comment:
If you're making something for the borgs, this needs to be a subtype; this is impacting the balance of existing immolators.